### ISSUE TITLE
"Browse subfolders" not visible in filepicker

### ISSUE THREAD

**Issue by papillon6** [fabarea/media#138]
See attached image for details.
TYPO3 7.6, media 4.0.4 

![bug_media](https://cloud.githubusercontent.com/assets/10079852/16188163/773acd74-36d4-11e6-90da-a7db7847bcdb.png)

**Comment by fabarea** [comment]
No, unfortunately it is much work to display a folder tree in the file picker. The "browser subfolder" is implicitly set.

**Comment by papillon6** [comment]
Hi Fab!

Although this issue is quite old I like to re-open it. Setting the "browse subfolder" option implicitly is a good "workaround" for this problem. Unfortunately the checkbox is "unchecked" per default in our case... We still have to open settings, close settings window to make the "browse subfolders" checkbox visible and activate it to display any files in subfolders.

Did I miss something? (affected media version 4.5.0)

Regards!

**Comment by fabarea** [comment]
What would be the ideal solution for you?
Would you be able to provide a PR in case?

**Comment by papillon6** [comment]
The mentioned workaround would be sufficient, if it worked (pre-checked checkbox "browse subfolders"). 
A PR? Sure - depends on your amount of effort. Just drop me a line and I'll get in touch with our customer to clarify whether it's worth it for our purpose.

**Comment by fabarea** [comment]
I was more thinking of **you** providing a PR :roll_eyes:

**Comment by mkrappitz** [comment]
We experience the same issue still today in 2020. This is so broken, that as of now it makes EXT:media kind of useless in TYPO3 9 IMHO. Here are my findings:

- Crazy enough for admin BE users the recursive search works by default in the EXT:media file picker.
- But for editor BE users with no admin rights, you have to use the above workaround to enter settings, leave settings, then the recursive checkbox finally appears and the recursice search even works. But when you want to add a found picture to an content element by clicking it, the picture does not get inserted in the content element, but the orginal image url is opened in a new tab.
- Then I thought to not use the EXT:media file picker and go back to the standard file picker of TYPO3 9. But the standard file picker ist then not searching in meta data any more, until you uninstall EXT:vidi and EXT:media.
- So the only current "solution" for now, unless you don't want to search in subfolders, is changing all your editors to admins, which is also nonsense.

All tested on TYPO3 9.5.13 and media 5.1.0-dev from master branch of this github repo.
